app/src/main/java/com/cornellappdev/android/eateryblue/ui/components/home/EateryHourBottomSheet.kt

```diff
@@ -0,0 +1,159 @@
+import com.cornellappdev.android.eateryblue.ui.theme.EateryBlueTypography
+import com.cornellappdev.android.eateryblue.ui.theme.GrayFive
+import com.cornellappdev.android.eateryblue.ui.theme.GrayOne
+import com.cornellappdev.android.eateryblue.ui.theme.GrayZero
+import com.cornellappdev.android.eateryblue.ui.theme.Green
+import com.cornellappdev.android.eateryblue.ui.theme.Red
+
+@OptIn(ExperimentalMaterialApi::class)
+@Composable
+fun EateryHourBottomSheet(
+    eatery: Eatery,
+    onDismiss: () -> Unit,
+    onReportIssue: () -> Unit
+) {
+    Column(
+        modifier = Modifier
+            .fillMaxWidth()
+            .padding(top = 14.dp)
+            .padding(16.dp),
+        horizontalAlignment = Alignment.Start
+    ) {
+        Row(
+            modifier = Modifier.fillMaxWidth(),
+            horizontalArrangement = Arrangement.SpaceBetween
+        ) {
+            Row {
+                Icon(
+                    imageVector = Icons.Outlined.Schedule,
+                    contentDescription = "Hours Icon",
+                    tint = Color.Black,
+                    modifier = Modifier.size(32.dp)
+                )
+                Spacer(modifier = Modifier.size(ButtonDefaults.IconSpacing))
+                androidx.compose.material.Text(
+                    text = "Hours",
+                    style = EateryBlueTypography.h4,
+                    color = Color.Black
+                )
+            }
+            IconButton(
+                onClick = {
+                    onDismiss()
+                },
+                modifier = Modifier
+                    .size(40.dp)
+                    .background(color = GrayZero, shape = CircleShape)
+            ) {
+                Icon(
+                    Icons.Default.Close,
+                    contentDescription = Icons.Default.Close.name,
+                    tint = Color.Black
+                )
+            }
+        }
+
+        Spacer(modifier = Modifier.height(12.dp))
+
+        val openUntil = eatery.getOpenUntil()
+        androidx.compose.material.Text(
+            modifier = Modifier.padding(top = 2.dp),
+            text = openUntil ?: "Closed",
+            style = TextStyle(
+                fontWeight = FontWeight.SemiBold, fontSize = 16.sp
+            ),
+            color = if (openUntil.isNullOrBlank()) Red else Green,
+        )
+
+        Spacer(modifier = Modifier.height(12.dp))
+
+        val operatingHours = eatery.formatOperatingHours()
+
+        operatingHours.forEachIndexed { index, (dayRange, hours) ->
+            Column {
+                Text(text = dayRange, fontSize = 16.sp, color = GrayFive)
+                hours.forEach { hour ->
+                    Text(
+                        text = hour,
+                        fontSize = 18.sp,
+                        color = Color.Black,
+                        fontWeight = FontWeight.Normal
+                    )
+                }
```

Comment:
There's a very slight discrepancy in font weights here. The little subheaders should have font weights of 500, and the actual hours have 600. There are constants for this, or you can just use `FontWeight(x)` where `x` is the font weight you want to put. Make sure to check little details like this on the Figma!

![image](https://github.com/cuappdev/eatery-blue-android/assets/47724806/b58b0127-d831-4c0a-8d60-511bb6efc7d1)